mu-trees/tests/unit/module-registries/requirejs-test.js

```diff
@@ -51,32 +53,73 @@ function buildMockRequire() {
 module('RequireJS Registry', {
   beforeEach() {
     this.mockRequire = buildMockRequire();
+    this.mockRequire.entries = {};
     this.config = config;
     this.registry = new RequireJSRegistry(this.config, 'src', this.mockRequire);
+  },
+
+  addModule(name, module) {
+    this.mockRequire.entries[name] = module;
   }
+
 });
 
-test('Normalize', function(assert) {
-  assert.expect(10);
+test('basic get', function(assert) {
+  assert.expect(8);
 
   [
+    /*
+     * Over time lets move these general cases into specific tests that
+     * describe their aim.
+     */
     [ 'router:/my-app/main/main', 'my-app/src/router' ],
     [ 'route:/my-app/routes/application', 'my-app/src/ui/routes/application/route' ],
     [ 'template:/my-app/routes/application', 'my-app/src/ui/routes/application/template' ],
     [ 'component:/my-app/components/my-input', 'my-app/src/ui/components/my-input/component' ],
     [ 'template:/my-app/routes/components/my-input', 'my-app/src/ui/components/my-input/template' ],
     [ 'template:/my-app/components/my-input', 'my-app/src/ui/components/my-input/template' ],
     [ 'component:/my-app/components/my-input/my-button', 'my-app/src/ui/components/my-input/my-button/component' ],
-    [ 'template:/my-app/components/my-input/my-button', 'my-app/src/ui/components/my-input/my-button/template' ],
-    [ 'template:/my-app/routes/-author', 'my-app/src/ui/partials/author' ],
```

Comment:
Seems like the other changes in this PR make partials still work, why remove the test?